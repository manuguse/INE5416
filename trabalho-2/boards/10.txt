5 - 2 - 2 - 3 1 3 1
- 4 - 1 - 5 - 5 - 4
7 5 1 7 - - 3 1 3 -
- 4 - - - - - - - 3
2 - 3 4 - 2 - - 4 -
5 - 2 - 6 - - - - -
- 1 3 - 1 - - 4 - 3
6 7 - 3 - 1 4 - - 1
4 - 3 - 4 - - - - 3
- 1 - 2 - 6 2 - 2 1