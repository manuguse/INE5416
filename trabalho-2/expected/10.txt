5 3 2 4 2 4 3 1 3 1
2 4 3 1 3 5 6 5 2 4
7 5 1 7 1 2 3 1 3 2
6 4 2 6 4 1 2 4 1 3
2 1 3 4 3 2 1 2 4 1
5 6 2 5 6 1 2 1 2 4
4 1 3 4 1 3 5 4 1 3
6 7 2 3 2 1 4 3 2 1
4 2 3 5 4 7 3 2 1 3
3 1 5 2 1 6 2 1 2 1